packages/db/package.json

```diff
@@ -2,9 +2,21 @@
   "name": "@dotkomonline/db",
   "version": "1.0.0",
   "type": "module",
-  "main": "./src/index.ts",
-  "types": "./src/index.ts",
   "private": true,
+  "exports": {
```

Comment:
Needed, so that `types` only imports schemas, and none of the actual Prisma client stuff. The prisma client breaks on Vercel (and shouldn't be used there for any reason either).

Same goes for the test harness